lib/modules/manager/bazel-module/readme.md

```diff
@@ -0,0 +1,6 @@
+<!-- prettier-ignore -->
+!!! warning
+    The `bazel-module` manager is work-in-progress.
+    For more information, see [issue 13658](https://github.com/renovatebot/renovate/issues/13658).
+
+The `bazel-module` manager supports the maintenance of [Bazel module (bzlmod)](https://bazel.build/external/module) enabled workspaces.
```

Comment:
Does the new feature support all things that people can put in a `bzlmod` file that are relevant here. Or are there missing things?

I'm basically asking if we need to expand the docs with "known limitations". 😉

Reply:
I can add sentence stating what is supported.